template/README.md

```diff
@@ -0,0 +1,52 @@
+# Getting Started with Create React App
```

Comment:
this readme may be replaced with the one we already have in 11.header of
[https://github.com/Hipo/web-handbook/blob/master/setting-up-a-new-project.md](url)

Reply:
Good idea, updated 👍